extensions/mssql/src/objectManagement/ui/databasePropertiesDialog.ts

```diff
@@ -0,0 +1,123 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import * as localizedConstants from '../localizedConstants';
+import { IObjectManagementService, ObjectManagement } from 'mssql';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { AlterUserDocUrl, CreateUserDocUrl } from '../constants';
+
+export class DatabasePropertiesDialog extends ObjectManagementDialogBase<ObjectManagement.DatabasePropertiesInfo, ObjectManagement.DatabasePropertiesViewInfo> {
+
+	// Horizontal Tabs
+	private generalTab: azdata.window.DialogTab;
+	private filesTab: azdata.window.DialogTab;
+
+	// GeneralTab: Backup section and its inputs
+	private backupSection: azdata.GroupContainer;
+	private lastDatabaseBackupInput: azdata.InputBoxComponent;
+	private lastDatabaseLogBackupInput: azdata.InputBoxComponent;
+
+	// GeneralTab: Database section and its inputs
+	private databaseSection: azdata.GroupContainer;
+	private nameInput: azdata.InputBoxComponent;
+	private statusInput: azdata.InputBoxComponent;
+	private ownerInput: azdata.InputBoxComponent;
+	private dateCreatedInput: azdata.InputBoxComponent;
+	private sizeInput: azdata.InputBoxComponent;
+	private spaceAvailabeInput: azdata.InputBoxComponent;
+	private numberOfUsersInput: azdata.InputBoxComponent;
+	private memoryAllocatedInput: azdata.InputBoxComponent;
+	private memoryUsedInput: azdata.InputBoxComponent;
+
+	// GeneralTab: Maintenance section and its inputs
+	private maintenanceSection: azdata.GroupContainer;
+	private collationInput: azdata.InputBoxComponent;
+
+	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
+		super(objectManagementService, options);
+		this.generalTab = azdata.window.createTab(localizedConstants.GeneralSectionHeader);
+		this.filesTab = azdata.window.createTab(localizedConstants.FilesHeaderText);
+	}
+
+	protected async initializeUI(): Promise<void> {
+		this.initializeBackupSection();
+		this.initializeDatabaseSection();
+		this.initializeMaintenanceSection();
+
+		this.registerTab(this.generalTab, [this.backupSection, this.databaseSection, this.maintenanceSection]);
+
+		this.dialogObject.content = [this.generalTab, this.filesTab];
+
+		// We need to close the already opened dialog during handleObjectPropertiesDialogCommand,
```

Comment:
I would suggest we fix this to not happen instead - otherwise we're going to have it blink up the "other" dialog briefly before closing and reopening (which will be especially bad for screen readers). Talk to @alanrenmsft if you need help with this

Reply:
@ssreerama Can you work with @alanrenmsft and do a fix for this separately? We definitely don't want to have this be included in this PR and it would be good to get that out first

Reply:
It has been fixed! Using tabbedPanel Instead of DailogTabs.